noxfile.py

```diff
@@ -11,7 +12,10 @@
 SILENT = VERBOSE == "0"
 
 # Linted dirs/files:
-targets = "."
+lint_targets = "."
+test_targets = [
+    "./" + t for t in find_namespace_packages(".") if ("configs" in t and "." not in t)
+]  # finds all config packages
```

Comment:
I would create an explicit array for the projects instead of discovering.
no need to get fancy here, we wouldn't be getting projects that often. (this might detect things we did not intend).

Reply:
Sure thing. We'll just have to make sure we update the array when we add a new project.